packages/flutter/lib/src/painting/text_painter.dart

```diff
@@ -253,10 +253,15 @@ class TextPainter {
     if (_text == value)
       return;
     _text = value;
-    ui.ParagraphBuilder builder = new ui.ParagraphBuilder();
-    _text.build(builder);
-    _paragraph = builder.build(_text.style?.paragraphStyle ?? new ui.ParagraphStyle());
-    _needsLayout = true;
+    if (_text != null) {
+      ui.ParagraphBuilder builder = new ui.ParagraphBuilder();
+      _text.build(builder);
+      _paragraph = builder.build(_text.style?.paragraphStyle ?? new ui.ParagraphStyle());
+      _needsLayout = true;
+    } else {
+      _paragraph = null;
```

Comment:
this will cause all the accessors to null-deref. none of them check whether _paragraph is set or not.